source/openapi-admin-v3.yaml

```diff
@@ -1317,7 +1321,7 @@ paths:
         - users
       operationId: adminCreateUser
       summary: Create a user
-      description: Create a [user](https://www.mongodb.com/docs/realm/authentication#std-label-user-accounts).
+      description: Create a new [user account](https://www.mongodb.com/docs/atlas/app-services/authentication#std-label-user-accounts).
```

Comment:
Should we add a note here that you can only create a user account for an email/password user, since other auth providers automatically create a user account upon validating a unique identity?